README.md

```diff
@@ -1 +1,139 @@
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is created by HTML and CSS to be showed a green colored Hello Microverse Title.
+
+
+## 🛠 Built With <a name="built-with"></a>
+Html and CSS
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[CSS file is externally linked to HTML]**
+- **[h1 tag is colored to green]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Code editor(VS), Web browser and Git.
+
+
+### Setup
+
+Clone this repository to your desired folder: https://github.com/kakalygaday/Hello-Microverse
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command: 
+Open the file with your default brower or use live server extension
+
+Example command:
+
+```sh
+  rails server
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdy**
+
+- GitHub: [@githubhandle](https://github.com/kakalygaday)
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Change the color of h1 tag]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project put a star and follow me for more.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank for these extra busy days.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Please remove the placeholder text from your `README` file because it is just for developer guidance purposes while producing the documentation of the project not for the public to read, and it looks unprofessional too.